src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_params.py

```diff
@@ -68,6 +68,7 @@
 register_cli_argument('group deployment', 'mode', help='Incremental (only add resources to resource group) or Complete (remove extra resources from resource group)', **enum_choice_list(DeploymentMode))
 register_cli_argument('group deployment create', 'deployment_name', options_list=('--name', '-n'), required=False,
                       validator=validate_deployment_name, help='The deployment name. Default to template file base name')
+register_cli_argument('group deployment operation show', 'operation_ids', nargs='+', required=True, help='A list of operation ids to show')
```

Comment:
You can remove the `required=True,` combining with the change I suggested below in `custom.py`. CLI command will recognize it is a required argument automatically